feature24/new_test.txt

```diff
@@ -0,0 +1,2 @@
+This file is a simple practice of the pull-request on the public repository. 
```

Comment:
**Great job, soon you will become a GitHub master!!!**

Reply:
Thanks for the approval :-)